cromshell

```diff
@@ -402,12 +405,178 @@ function populateWorkflowIdAndServerUrl()
   fi
 }
 
+function _turtleSpinner()
+{
+  # Require a flag file to be given to this function.
+  # This keeps it safe.
+  if [[ $# -ne 1 ]] ; then
+    return 1
+  fi
+
+  local flagFile=$1
+
+  # Let's be fancy and make turtles that are walking while we wait:
+  local tmpTurtFile=$( makeTemp )
+  local turtFile=$( makeTemp )
```

Comment:
will these be cleaned up when this spun off thread dies or when the main one does?  it doesn't matter which it is, I'm just curious how inheriting traps works.

Reply:
The trap is for the script as a whole, so they'll only be deleted when the main script itself exits.